pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -77,6 +77,10 @@ type agentClusterInstallOnPremPlatform struct {
 	// ProvisioningDHCPRange is used to provide DHCP services to hosts
 	// for provisioning.
 	ProvisioningDHCPRange string `json:"provisioningDHCPRange,omitempty"`
+
+	// AdditionalNTPServers defines a list of additional NTP
+	// servers to use for provisioning
+	AdditionalNTPServers []string `json:"additionalNTPServers,omitempty"`
```

Comment:
Since assisted-service already has the list of additionalNTPServers, it would be better to have it always insert those into the install-config it generates (which would solve the problem for all types of assisted installations) rather than add it to the install-config overrides here (which solves it only for ABI). I suggest opening a ticket for that.

If we're going to do this as an interim measure, then just adding a field in this struct is not enough, there has to be some code in this file to populate it.

Reply:
No, this is additional to that. It's about passing through the config to populate in clusters created by assisted for use on day 2.